packages/core/src/utils/markers.ts

```diff
@@ -0,0 +1,108 @@
+import Style from 'ol/style/Style'
+import Icon from 'ol/style/Icon'
+
+export interface MarkerStyle {
+  strokeWidth?: string | number
+  stroke?: string
+  fill?: string
+}
+
+export type GetMarkerFunction = (style: MarkerStyle, multi: boolean) => Style
+
+export const imgSize = [23, 32]
+export const imgSizeMulti = [35, 32]
+
+export const defaultStroke = '#FFFFFF'
+export const defaultStrokeWidth = '2'
+
+export const defaultFill = '#005CA9'
+export const defaultHoverFill = '#7B1045'
+export const defaultSelectionFill = '#679100'
+
+const prefix = 'data:image/svg+xml,'
+
+/* Path of marker svg used in this file copied and adapted from
+ * @masterportal/masterportalapi/public/marker.svg. */
+
+export const makeMarker = ({
+  stroke = defaultStroke,
+  strokeWidth = defaultStrokeWidth,
+  fill = defaultFill,
+}: MarkerStyle) =>
+  `${prefix}${encodeURIComponent(`
+<svg width="${imgSize[0]}" height="${imgSize[1]}" viewBox="0 0 30 43" xmlns="http://www.w3.org/2000/svg">
+  <title>DB6C494E-88E8-49F1-89CE-97CBEC3A5240</title>
+  <path
+    d="M14.584 1C7.101 1 1 7.101 1 14.584c0 8.103 7.865 20.448 11.63 25.93a2.36 2.36 0 0 0 3.908 0c3.766-5.482 11.63-17.922 11.63-25.93C28.168 7.054 22.068 1 14.584 1z"
+    stroke="${stroke}"
+    stroke-width="${strokeWidth}"
+    fill="${fill}"
+    fill-rule="nonzero"
+  />
+</svg>
+`)}`
+
+export const makeMultiMarker = ({
+  stroke = defaultStroke,
+  strokeWidth = defaultStrokeWidth,
+  fill = defaultFill,
+}: MarkerStyle) =>
+  `${prefix}${encodeURIComponent(`
+<svg width="${imgSizeMulti[0]}" height="${imgSizeMulti[1]}" viewBox="0 0 30 43" xmlns="http://www.w3.org/2000/svg">
+  <title>0A6F4952-4A5A-4E86-88E4-4B3D2EA1E3DF</title>
+  <g stroke="${stroke}" stroke-width="${strokeWidth}" fill="${fill}" fill-rule="nonzero">
+    <path d="M22.584 1C15.101 1 9 7.101 9 14.584c0 8.103 7.865 20.448 11.63 25.93a2.36 2.36 0 0 0 3.908 0c3.766-5.482 11.63-17.922 11.63-25.93C36.168 7.054 30.068 1 22.584 1z"/>
+    <path d="M18.584 1C11.101 1 5 7.101 5 14.584c0 8.103 7.865 20.448 11.63 25.93a2.36 2.36 0 0 0 3.908 0c3.766-5.482 11.63-17.922 11.63-25.93C32.168 7.054 26.068 1 18.584 1z"/>
+    <path d="M14.584 1C7.101 1 1 7.101 1 14.584c0 8.103 7.865 20.448 11.63 25.93a2.36 2.36 0 0 0 3.908 0c3.766-5.482 11.63-17.922 11.63-25.93C28.168 7.054 22.068 1 14.584 1z"/>
+  </g>
+</svg>
+`)}`
+
+// center bottom of marker 📍 is intended to show the spot
+export const anchor = [0.5, 1]
```

Comment:
All of these exported `const` are never used outside of this file.

Is there an intended use for them that is not clear yet?

Reply:
I figured that having these values available could help with re-use of the marker designs later by allowing to use lines like `import { makeMarker } from '@polar/core/src/utils/markers'` in a client.

Reply:
Hm, I think we never really decided on it, but wouldn't that fit more into a lib-package then?

Reply:
https://github.com/Dataport/polar/pull/42/commits/c799e39e59471fda0baf3a2357cc45cc10aa09af

Narrowed it for now.